packages/@guardian/source-react-components/src/link/types.ts

```diff
@@ -10,10 +10,7 @@ export interface SharedLinkProps extends Props {
 	 * Informs users of how important a link is
 	 */
 	priority?: LinkPriority;
-	/**
-	 * Whether link is subdued (no underline)
-	 */
-	subdued?: boolean;
```

Comment:
Given there are a [number of places using this](https://github.com/search?p=3&q=org%3Aguardian+subdued+-repo%3Aguardian%2Fsource+-repo%3Aguardian%2Fsource-dac+-repo%3Aguardian%2Fsource-shakedown&type=Code), I wonder if we should deprecate the prop rather than delete it? That way we don't introduce a breaking API change in this version.

What do you think?

Reply:
Good idea!